microsoft-edge/webview2/concepts/userdatafolder.md

```diff
@@ -0,0 +1,62 @@
+
+## Create user data folders
+
+To specify the location of the user data folder, include the `userDataFolder` parameter when calling `CoreWebView2Environment`. After creation, browser data from your WebView2 control is stored in a subfolder of `userDataFolder`. When `userDataFolder` is not specified, WebView2 creates user data folders at default locations as follows:
+
+* For packaged Windows Store apps, the default user folder is the `ApplicationData\LocalFolder` subfolder in the package’s  folder.
+* For existing desktop apps, the default user data folder is the exe path of your application + `.WebView2`. Instead of using the default, we recommend that you specify a user data folder, and that you create it in the same folder where all other app data is stored.
+
+## Delete user data folders
+
+Your application may need to delete user data folders:
+
+* When uninstalling your app. If you are uninstalling packaged Windows Store apps, Windows deletes user data folders automatically. 
+* To clean up all browsing data history.
+* To recover from data corruption.
+* To remove previous session data. 
+
+
+> [!NOTE]
+> Files in user data folders may still be in use after the WebView2 application is closed. In this situation, wait for the browser process and all child processes to exit before deleting the folder. You may retrieve the process id of the browser process using the `BrowserProcessId` property of the WebView2.
+
+## Share user data folders
+
+WebView2 controls may share the same user data folders to:
+
+* Optimize system resources by running in one browser process.
+* Share browser history and cached resources. 
+
+Consider the following when sharing user data folders: 
+
+1. When re-creating WebView2 controls to update browser versions using `NewBrowserVersionAvailable` events, ensure browser processes exit and close WebView2 controls that share the same user data folder. To retrieve the process id of the browser process, use the `BrowserProcessId` property of the WebView2 control.
+
+2. WebView2s sharing the same user data folder must use the same options for **CoreWebView2Environment**. If not, the WebView2 creation will fail with `HRESULT_FROM_WIN32(ERROR_INVALID_STATE)`. For example, if the first WebView2 is created by **CoreWebView2Environment** with a certain **AdditionalBrowserArguments**, trying to create WebView2 from **CoreWebView2Environment** with different **AdditionalBrowserArguments** will fail.
+
+To isolate different parts of your application or when sharing data between WebView2 controls is not needed, you may choose to use different user data folders. For example, an application may consist of two WebView2 controls, one for displaying ads and the other for displaying application content. In this scenario, developers may opt to use different user data folders for each WebView2 control. 
+
+> [!NOTE]
```

Comment:
Please add a See Also section with some related links. Thanks.

Reply:
What links do you recommend?

Reply:
@pagoe-msft - Can we link to the getting started topics?